src/themes/ec/_variables.scss

```diff
@@ -573,6 +573,29 @@ $list: (
   divider-color: var(--c-n),
 );
 
+// List with illustrations
+$list-illustration: (
+  title-color: var(--c-d),
+  description-color: var(--c-d),
+  background-color: var(--c-n-40),
+  value-font-mobile: var(--f-2xl),
+  title-font: var(--f-l),
+  value-font-weight-mobile: map.get($font-weight, 'regular'),
+  value-font-desktop: var(--f-3xl),
+  value-font-weight-desktop: map.get($font-weight, 'regular'),
+  icon-color: var(--c-d),
+);
+$list-illustration-print: (
+  title-color: var(--c-d),
```

Comment:
just a remak: if the colors are the same on print, we could use the variables defined for the screen version, no need to have a specific variable here.
Usually, the only variable that I define for print are for spacing and typography as they are different from screen display